lib/std/heap/memory_pool.zig

```diff
@@ -32,6 +35,10 @@ pub const Options = struct {
 };
 
 /// A memory pool that can allocate objects of a single type very quickly.
+/// This implementation allows both specifying a user-defined item alignment
+/// and the option to prevent future allocations after the initialization.
+/// This is especially useful in memory-constraint applications.
```

Comment:
Just trying to be helpful, so please tell me if I am not:

"memory-constraint" => "memory-constrained"